mlflow/R/mlflow/R/tracking-runs.R

```diff
@@ -427,7 +427,7 @@ mlflow_download_artifacts <- function(path, run_id = NULL, client = NULL) {
 # ' Download Artifacts from URI.
 mlflow_download_artifacts_from_uri <- function(artifact_uri, client = mlflow_client()) {
   result <- mlflow_cli("artifacts", "download", "-u", artifact_uri, echo = FALSE, client = client)
-  gsub("\n", "", result$stdout)
+  gsub("[\r\n]", "", result$stdout)
```

Comment:
Can we use `trimws()` here?

https://stat.ethz.ch/R-manual/R-patched/library/base/html/trimws.html

Reply:
Yes, good shout. Looks like the correction here `[\r\n]` fixes the broken tests. So `trimws()` should also do the same job. I'll push a change now.